docs/platforms/python/crons/index.mdx

```diff
@@ -12,6 +12,70 @@ Sentry Crons allows you to monitor the uptime and performance of any scheduled,
 
 <PlatformContent includePath="crons/setup" />
 
+## Check-Ins
+
+Check-in monitoring allows you to track a job's progress by capturing two check-ins: one at the start of your job and another at the end of your job. This two-step process allows Sentry to notify you if your job didn't start when expected (missed) or if it exceeded its maximum runtime (failed).
+
+If you use the `monitor` decorator/context manager, the SDK will create check-ins for the wrapped code automatically.
+
+```python
+from sentry_sdk.crons import capture_checkin
+from sentry_sdk.crons.consts import MonitorStatus
+
+check_in_id = capture_checkin(
+    monitor_slug='<monitor-slug>',
+    status=MonitorStatus.IN_PROGRESS,
+)
+
+# Execute your task here...
+
+capture_checkin(
+    monitor_slug='<monitor-slug>',
+    check_in_id=check_in_id,
+    status=MonitorStatus.OK,
+)
+```
+
+## Upserting Cron Monitors
```

Comment:
I would avoid the word "upsert" and rather call this section: "Adding cron monitor configuration options" or something like this. (and later in the text mention that when you add the monitor_config and no monitor is existing it is created.)

Reply:
Changed to `Configuring Cron Monitors` and added a sentence: `If a monitor with the defined settings doesn't exist in Sentry yet, it will be created.`